src-tauri/tauri.conf.json

```diff
@@ -73,6 +73,6 @@
       "open": true
     }
   },
-  "productName": "Zoo Modeling App",
+  "productName": "zoo-modeling-app",
```

Comment:
@lf94 Can you check how this looks on Windows and macOS, aka platforms that would automatically Title Case the names? I would prefer not to change how those look (and it could break the updater too)

Reply:
Reverted